package mpsqual;

message RequestStatusMessage {
	required string name = 1;
}

message ResponseStatusMessage {
	enum ErrStatusCode {
		STATUS_OK = 0;
		STATUS_INVALID_NAME = 1;
		STATUS_SEMA_ACCESS_ERROR = 2;
		STATUS_IMPROPER_MESSAGE = 4;
		STATUS_UNKNOWN_ERR = 6;
    }
	
	required string name = 1;
	required ErrStatusCode error = 2;
	optional string value = 3;
}

